app/models/file_search.rb

```diff
@@ -112,6 +120,20 @@ def filters
       .merge(work_filter)
       .merge(venue_filter)
       .merge(year_filter)
+      .merge(resource_type_filter)
+      .merge(highlight_filter)
+  end
+
+  def highlight_filter
+    # 'highlighted' is only used on first view of index
+    return {} if has_query_params?
+    { Solrizer.solr_name("highlighted", :facetable) => "1" }
+  end
+
+  def resource_type_filter
+    return {} if resource_types.empty? || resource_types == all_resource_types
```

Comment:
Is `resource_types` used anywhere but here?  If not, perhaps this could be simplified and the `resource_types` method eliminated.  Something like:

```ruby
return {} unless resource_type.present?
{ Solrizer.solr_name("resource_type", :facetable) => Array(resource_type.singularize.capitalize) }
```